submissions/EmmanuelIdiaghe/css/signup-page.html

```diff
@@ -11,9 +11,9 @@
         <img src="images/EMANCIPATOR.png" alt="logo" class="logo">
         <nav>
```

Comment:
@emmanuelidiaghe html files are suppose to be in html folder. You should move this file alongside other html files in the css folder to the html folder.